src/pages/Regions/RegionEdit.tsx

```diff
@@ -99,32 +100,51 @@ const RegionEditFormPage = () => {
   };
 
   const handleSubmit = async (values: any) => {
-    const newRegion: any = {
-      regionName: values.regionName,
-      description: values.description,
-      phoneNumber: values.phoneNumber,
-      email: values.email,
-      link: values.link,
-      street: values.street,
-      houseNumber: values.houseNumber,
-      officeNumber: values.officeNumber,
-      postIndex: values.postIndex,
-      logo: logo,
-      city: values.city,
-      isActive: chosenRegion.isActive
-    };
-    await RegionsApi.EditRegion(currentRegion, newRegion);
+    setLoading(true);
 
-    form.resetFields();
+    const checkResponce = await checkIfNameExists(values.regionName);
```

Comment:
Does it really check some response, I guess it is checking just `regionName` for existing
So rename it please to something more appropriate

Reply:
Done!